core/sourceCode.ml

```diff
@@ -10,91 +10,205 @@ let trim_initial_newline s =
   else s
 
 class source_code =
-object (self)
-  val lines =
-    let tbl = Hashtbl.create default_lines in
+  object (self)
```

Comment:
I think you should undo this change as it only introduces indentation. I don't think we have a coding convention for indentation at the moment, but it would be a silly if future commits went back and forth about the indentation level.

Reply:
Yeah I can undo this, this was accidentally/automatically done by merlin. I think some day it would possibly be nice to find a coding convention for this and talk about it at the next links meeting?

Reply:
If anything, I think it is worth to separate large formatting changes into separate commits.

And personally I don't mind this extra indentation. I think this is how Merlin's indenter would format it, right?

Reply:
I agree, it would be good to settle on a convention, but as we don't have one at the moment (at least I am not aware of one) I think we should avoid introducing such "trivial" changes.

Reply:
@jstolarek yes this is merlin. I'm happy for this to be farmed out to a seperate commit, again this was more accidental than on purpose.